### PR TITLE
Issue #1770: Update missing translation key and add FAT

### PR DESCRIPTION
**Pull request by pnicolucci**

### CODE REVIEW

**Comment by pnicolucci** [comment]
fixes #1770

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.4.0_fat

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
@OpenLiberty/message-reviewer

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops [hunk 1]
```diff
@@ -1585,6 +1585,15 @@ uncaught.exception.during.AsyncListener.onComplete.useraction=Verify the AsyncLi
...
+
 #name.is.null=CWWWC0400E: An attribute name, or initial parameter name, is null.
 #name.is.null.explanation=This is an application error.  The name is null in either context getAttribute(), setAttribute(), getInitParameter(), or setInitParameter().
 #name.is.null.useraction=Update the attribute name, or initial parameter name, with a non-null value.
+
+# Notes for translation
+# {0} is the name of the ServletContainerInitializer that was being invoked at the time of the Exception.
+# {1} is the Exception that was thrown from the onStartup method of the ServletContainerInitializer {0}.
+# {2} is the application name that the ServletContainerInitializer is contained in.
+exception.occurred.while.running.ServletContainerInitializers.onStartup=CWWWC0001W: ServletContainerInitializer [{0}] threw the following exception while invoking the onStartup method: [{1}] located in the following application: [{2}]
```
[the remark this answers is not in the document]
Comment (pnicolucci) [reply dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: @dmuelle  the ServletContainerInitializer is located within the application. [{0}]

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops [hunk 2]
```diff
@@ -1585,6 +1585,15 @@ uncaught.exception.during.AsyncListener.onComplete.useraction=Verify the AsyncLi
...
+exception.occurred.while.running.ServletContainerInitializers.onStartup.explanation=An exception was thrown from a ServletContainerInitializer while invoking the onStartup method.
```
[the remark this answers is not in the document]
Comment (pnicolucci) [reply dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: @dmuelle, unfortunately we don't know what Exception or Error the application code might / could throw. So honestly this comes back to the app and the customer needs to look at their application / logs and determine what Exception their application threw.

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmuelle) [reply dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: ok- in that case
The [{0}] ServletContainerInitializer located in the following application [{2}] produced the following error while invoking the onStartup method: [{1}]

**Comment by andrea-pichler** [comment]
No message review requested from L2

**Comment by pnicolucci** [comment]
@dmuelle I've made the requested changes, can you please review? Thanks!

**Comment by pnicolucci** [comment]
#build

**Pull request merged**